azurerm/resource_arm_redis_cache.go

```diff
@@ -101,6 +102,28 @@ func resourceArmRedisCache() *schema.Resource {
 							Default:      "volatile-lru",
 							ValidateFunc: validateRedisMaxMemoryPolicy,
 						},
+						"rdb_backup_enabled": {
+							Type:             schema.TypeString,
+							Optional:         true,
+							DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
+							ValidateFunc: validation.StringInSlice([]string{
```

Comment:
Is there any particular reason we want to have string here? Stringifyed boolean values don't look nice and break the whole point of having a (semi)typed schema IMO.

Even if the reason here is API then I think we should solve that problem and do our bit internally in the CRUD and/or `DiffSuppressFunc`/`StateFunc` so that the user doesn't have to deal with it.